### DISCUSSION TITLE
version

### DISCUSSION THREAD

**Discussion by kittu1999** [bentoml/BentoML#2408] [General]
I started with bentoml recently, i am facing lot of issues with new version. Can you tell me which version can i use?

**Comment by timliubentoml** [comment]
I would suggest using version a7. Is that the version you're using? What issues are you having?

**Reply by kittu1999** [reply]
I am using a6 version. In the  performance tracing i am getting this error "No module named 'opentelemetry.exporter'"

**Reply by parano** [reply]
@kittu1999 what's your opentelemetry-api and opentelemetry-sdk version? you can find the version with `pip freeze | grep opentelemetry`

**Reply by kittu1999** [reply]
opentelemetry-sdk==1.9.0,  opentelemetry-api==1.9.0

**Reply by parano** [reply]
@kittu1999 that looks right to me, could you try it again with a7 version? cc @bojiang

**Comment by parano** [comment]
There are currently two active BentoML versions: version 0.13 and version 1.0.0 preview releases(1.0.0a7 is the latest). 0.13 is a stable release and you can find the source code in the 0.13-LTS branch https://github.com/bentoml/BentoML/tree/0.13-LTS and documentation at https://docs.bentoml.org/en/0.13-lts/ - this version is what you get from `pip install bentoml`

1.0.0 is a new generation of BentoML that is currently under beta preview release and active development. It is what you see in the current default branch in the BentoML repo, you can find the docs here: https://docs.bentoml.org, and install it by providing the `--pre` flag, so `pip` will install the preview release: `pip install --pre bentoml`

1.0.0 comes with drastic changes in the design, and it is not backwards compatible.  If you are seeing lots of errors, it might be because you are trying to run code samples for 1.0, using BentoML version 0.13, or vise versa.  You can check which version you have installed with `bentoml --version` command.
